### PR TITLE
Ignore sm acknowledged errors

### PR DESCRIPTION
**Pull request by ostap0207**

### CODE REVIEW

File: src/Logger.js [hunk 1]
```diff
@@ -107,17 +107,21 @@ export default function Logger({
     };
   };
 
-  const includesError = args =>
-    args.filter(arg => arg instanceof Error).length > 0;
+  const findTopLevelError = args => args.filter(arg => arg instanceof Error)[0];
```
Comment (ostap0207) [inline src/Logger.js:110]: Checked if IE supports `find` and it doesn't, so changed to filter.

**Comment by ostap0207** [comment]
!merge

**Comment by indrekj** [comment]
Can we do this without sm prefix?

File: src/Logger.js (hunk 1 above)
Comment (urmastalimaa) [reply src/Logger.js:110]: Ah, I somehow completely blanked and thought this was client-logger (back end) and didn't notice this.

**Pull request merged**